include/eve/detail/function/simd/common/broadcast_group.hpp

```diff
@@ -0,0 +1,101 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/abi.hpp>
+#include <eve/pattern.hpp>
+
+namespace eve::detail
+{
+  template<simd_value Wide, std::ptrdiff_t Group, std::ptrdiff_t Index, std::ptrdiff_t Size>
+  requires(   (Group >  0) && (Group <= std::min(cardinal_v<Wide>, Size))
+          &&  (Index >= 0) && (Index < cardinal_v<Wide>/Group)
+          )
```

Comment:
I don't understand the min requirement. Size is always less than min.

Reply:
you can ask to broadcast the Nth block of 2 values from a 4 value wide into a 8 value one.
Turning  (a b c d) into (c d c d c d c d)